data/Viral kinetics of SARS-CoV-2/code.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import yaml
+import pandas as pd
+import numpy as np
+class folded_str(str): pass
+class literal_str(str): pass
+
+def folded_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='>')
+def literal_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+yaml.add_representer(folded_str, folded_str_representer)
+yaml.add_representer(literal_str, literal_str_representer)
+
+
+# In[2]:
+
+
+data = pd.read_csv("CombinedDataset-2.csv") 
+data=data[113:132]
+
+
+# In[4]:
+
+
+data = data.sort_values(by=['Day','StudyNum'])
+
+
+# In[5]:
+
+
+data = data.replace({"Sex": {"M": "male", "F": "female"},
+                           "SevMax3": {"Moderate": "Moderate", "Mild": "Mild", "Severe": "Severe"},
+                           "PatientID":{"4-1":"1", "4-2":"2","4-3":"3"}})
+print(data)
+
+
+# In[14]:
+
+
+df = pd.DataFrame(data) if not isinstance(data, pd.DataFrame) else data
+
+participants = []
+
+for patient_id in df["PatientID"].unique():
+    patient_data = df[df["PatientID"] == patient_id]
```

Comment:
```suggestion
for patient_id, patient_data in df.groupby("PatientID"):
```

This is a slightly easier way to group data and get subsets.